### ISSUE TITLE
[BUG]Bittrex API orderbook - WARNING - Snapshot query timed out. Retrying...

### ISSUE THREAD

**Issue by sweetpotatoz** [CoinAlpha/hummingbot#985]
**Describe the bug**
Log file as below.
Running XEMM (maker=Bittrex, taker=Binance)
version 0.17.1
strategy file attached as .txt below

```
2019-10-25 12:52:50,536 - 10 - hummingbot.market.binance.binance_market - INFO - Network status has changed to NetworkStatus.CONNECTED. Starting networking...
2019-10-25 12:52:50,648 - 10 - hummingbot.market.binance.binance_api_order_book_data_source - INFO - Initialized order book for XZCBTC. 1/1 completed.
2019-10-25 12:52:50,704 - 10 - hummingbot.market.binance.binance_api_user_stream_data_source - INFO - Reconnecting to wss://stream.binance.com:9443/ws/tu8O9xtJAdNaQIh3UPEvlUH9DBb2NPKj94epX0iD2ovlWSMQsdsn4498L8c1.
2019-10-25 12:52:51,615 - 10 - hummingbot.market.bittrex.bittrex_market - INFO - Network status has changed to NetworkStatus.CONNECTED. Starting networking...
2019-10-25 12:52:51,760 - 10 - hummingbot.market.bittrex.bittrex_api_order_book_data_source - INFO - Subscribed to XZC-BTC deltas
2019-10-25 12:52:51,760 - 10 - hummingbot.market.bittrex.bittrex_api_order_book_data_source - INFO - Query XZC-BTC snapshot.
2019-10-25 12:52:52,079 - 10 - hummingbot.market.bittrex.bittrex_api_order_book_data_source - INFO - Query XZC-BTC snapshots.[Scheduled]
2019-10-25 12:52:52,080 - 10 - hummingbot.market.bittrex.bittrex_api_order_book_data_source - INFO - Query XZC-BTC snapshot[1572007972]. 1/20
2019-10-25 12:52:52,080 - 10 - hummingbot.market.bittrex.bittrex_api_user_stream_data_source - INFO - Invoked GetAuthContext
2019-10-25 12:52:53,260 - 10 - hummingbot.market.binance.binance_order_book_tracker - INFO - Started order book tracking for XZCBTC.
2019-10-25 12:52:58,963 - 10 - hummingbot.market.bittrex.bittrex_api_order_book_data_source - WARNING - Snapshot query timed out. Retrying...
2019-10-25 12:52:59,464 - 10 - hummingbot.market.bittrex.bittrex_api_order_book_data_source - INFO - Query XZC-BTC snapshot[1572007979]. 8/20
2019-10-25 12:53:02,081 - 10 - hummingbot.market.bittrex.bittrex_api_order_book_data_source - WARNING - Snapshot query timed out. Retrying...
2019-10-25 12:53:02,582 - 10 - hummingbot.market.bittrex.bittrex_api_order_book_data_source - INFO - Query XZC-BTC snapshot[1572007982]. 2/20
2019-10-25 12:53:09,466 - 10 - hummingbot.market.bittrex.bittrex_api_order_book_data_source - WARNING - Snapshot query timed out. Retrying...
2019-10-25 12:53:09,968 - 10 - hummingbot.market.bittrex.bittrex_api_order_book_data_source - INFO - Query XZC-BTC snapshot[1572007989]. 9/20
2019-10-25 12:53:12,583 - 10 - hummingbot.market.bittrex.bittrex_api_order_book_data_source - WARNING - Snapshot query timed out. Retrying...
2019-10-25 12:53:13,085 - 10 - hummingbot.market.bittrex.bittrex_api_order_book_data_source - INFO - Query XZC-BTC snapshot[1572007993]. 3/20
2019-10-25 12:53:19,969 - 10 - hummingbot.market.bittrex.bittrex_api_order_book_data_source - WARNING - Snapshot query timed out. Retrying...
2019-10-25 12:53:20,470 - 10 - hummingbot.market.bittrex.bittrex_api_order_book_data_source - INFO - Query XZC-BTC snapshot[1572008000]. 10/20
2019-10-25 12:53:23,086 - 10 - hummingbot.market.bittrex.bittrex_api_order_book_data_source - WARNING - Snapshot query timed out. Retrying...
2019-10-25 12:53:23,587 - 10 - hummingbot.market.bittrex.bittrex_api_order_book_data_source - INFO - Query XZC-BTC snapshot[1572008003]. 4/20
2019-10-25 12:53:30,472 - 10 - hummingbot.market.bittrex.bittrex_api_order_book_data_source - WARNING - Snapshot query timed out. Retrying...
2019-10-25 12:53:30,974 - 10 - hummingbot.market.bittrex.bittrex_api_order_book_data_source - INFO - Query XZC-BTC snapshot[1572008010]. 11/20
2019-10-25 12:53:33,590 - 10 - hummingbot.market.bittrex.bittrex_api_order_book_data_source - WARNING - Snapshot query timed out. Retrying...
2019-10-25 12:53:34,092 - 10 - hummingbot.market.bittrex.bittrex_api_order_book_data_source - INFO - Query XZC-BTC snapshot[1572008014]. 5/20
2019-10-25 12:53:40,976 - 10 - hummingbot.market.bittrex.bittrex_api_order_book_data_source - WARNING - Snapshot query timed out. Retrying...
2019-10-25 12:53:41,477 - 10 - hummingbot.market.bittrex.bittrex_api_order_book_data_source - INFO - Query XZC-BTC snapshot[1572008021]. 12/20
2019-10-25 12:53:44,093 - 10 - hummingbot.market.bittrex.bittrex_api_order_book_data_source - WARNING - Snapshot query timed out. Retrying...
2019-10-25 12:53:44,594 - 10 - hummingbot.market.bittrex.bittrex_api_order_book_data_source - INFO - Query XZC-BTC snapshot[1572008024]. 6/20
2019-10-25 12:53:51,478 - 10 - hummingbot.market.bittrex.bittrex_api_order_book_data_source - WARNING - Snapshot query timed out. Retrying...
2019-10-25 12:53:51,979 - 10 - hummingbot.market.bittrex.bittrex_api_order_book_data_source - INFO - Query XZC-BTC snapshot[1572008031]. 13/20
2019-10-25 12:53:54,595 - 10 - hummingbot.market.bittrex.bittrex_api_order_book_data_source - WARNING - Snapshot query timed out. Retrying...
2019-10-25 12:53:55,098 - 10 - hummingbot.market.bittrex.bittrex_api_order_book_data_source - INFO - Query XZC-BTC snapshot[1572008035]. 7/20
2019-10-25 12:54:01,980 - 10 - hummingbot.market.bittrex.bittrex_api_order_book_data_source - WARNING - Snapshot query timed out. Retrying...
2019-10-25 12:54:02,482 - 10 - hummingbot.market.bittrex.bittrex_api_order_book_data_source - INFO - Query XZC-BTC snapshot[1572008042]. 14/20
2019-10-25 12:54:05,099 - 10 - hummingbot.market.bittrex.bittrex_api_order_book_data_source - WARNING - Snapshot query timed out. Retrying...
2019-10-25 12:54:05,601 - 10 - hummingbot.market.bittrex.bittrex_api_order_book_data_source - INFO - Query XZC-BTC snapshot[1572008045]. 8/20
2019-10-25 12:54:12,483 - 10 - hummingbot.market.bittrex.bittrex_api_order_book_data_source - WARNING - Snapshot query timed out. Retrying...
2019-10-25 12:54:12,985 - 10 - hummingbot.market.bittrex.bittrex_api_order_book_data_source - INFO - Query XZC-BTC snapshot[1572008052]. 15/20
2019-10-25 12:54:15,603 - 10 - hummingbot.market.bittrex.bittrex_api_order_book_data_source - WARNING - Snapshot query timed out. Retrying...
2019-10-25 12:54:16,105 - 10 - hummingbot.market.bittrex.bittrex_api_order_book_data_source - INFO - Query XZC-BTC snapshot[1572008056]. 9/20
2019-10-25 12:54:22,987 - 10 - hummingbot.market.bittrex.bittrex_api_order_book_data_source - WARNING - Snapshot query timed out. Retrying...
2019-10-25 12:54:23,490 - 10 - hummingbot.market.bittrex.bittrex_api_order_book_data_source - INFO - Query XZC-BTC snapshot[1572008063]. 16/20
2019-10-25 12:54:26,107 - 10 - hummingbot.market.bittrex.bittrex_api_order_book_data_source - WARNING - Snapshot query timed out. Retrying...
```

**Steps To Reproduce**
// A concise description of the **steps to reproduce the buggy behavior**:

**Screenshots**
// If applicable, add screenshots to help explain your problem.

**Release version**
// Include your bot's version number (Can be found at the bottom left corner of your CLI).
0.17.1

**Attachments**
// Please attach your config file and log file to help us debug. It would be difficult for us to help you without those! 

[XEMM_Binance_Bittrex_XZCBTC.txt](https://github.com/CoinAlpha/hummingbot/files/3772144/XEMM_Binance_Bittrex_XZCBTC.txt)

**Comment by rennel-tabing** [comment]
I am able to reproduce the error. I am attaching the logs from my bot

[logs_bittrexXemm.log](https://github.com/CoinAlpha/hummingbot/files/3772238/logs_bittrexXemm.log)

**Comment by CrimsonJacket** [comment]
@sweetpotatoz We are following Bittrex's v3 API market symbol standards. Looking at your configs, the `maker_market_symbol` is `BTC-XZC` which is still follows v1.1 convention. In this case, it should be `XZC-BTC`.
This issue would be fixed once PR #978 has been merged.
